draft-ietf-ppm-dap.md

```diff
@@ -519,6 +519,26 @@ DAP has three major interactions which need to be defined:
 * Computing the results of a given measurement, specified in {{aggregate-flow}}
 * Collecting aggregated results, specified in {{collect-flow}}
 
+In this section, we discuss each of the resources and the messages used to act
+on these resources. A resource's path is resolved relative to a server's
+endpoint to construct a resource URI. A server's endpoint MAY contain
+arbitrarily many path components before the DAP resources.
```

Comment:
This guidance doesn't seem necessary to me given the definition of `[role]` below.

```suggestion
endpoint to construct a resource URI.
```